.github/.workflows/deploy.yml

```diff
@@ -2,7 +2,11 @@ name: Deploy
 
 on:
   push:
-    branches: master
+    branches:
+      - main
+      - master
```

Comment:
Since this repo has main as default branch and master is not used, no need to specify master